### PR TITLE
TEAM 3 Refactored Seed and provided details

### PR DESCRIPTION
**Pull request by yatmchang**
Refactored DB:Seeds and provided details

### CODE REVIEW

**Comment by tkbeili** [comment]
Please rebase on top of integration

**Comment by yatmchang** [comment]
Please review.

**Comment by tkbeili** [comment]
@yatmchang I can't review without proper rebase, this PR still says: `This branch has conflicts that must be resolved`

File: db/seeds.rb [hunk 1]
```diff
@@ -5,162 +5,100 @@
...
+  }
+]
+organization_list.each do |org|
+  Organization.create( org )
+  puts "Creating #{org[:name]}"
+end
 
 ClaimRequest.create(user_id: User.first.id, organization_id: Organization.first.id)
 ClaimRequest.create(user_id: User.last.id, organization_id: Organization.last.id)
+puts "Created Claim Request"
```
Comment (tkbeili) [inline db/seeds.rb]: Minor fix, please say `Created 2 Claim Requests` Also add a print to display how many users and how many organizations have been created.

**Comment by yatmchang** [comment]
fixed

File: db/seeds.rb [hunk 2]
```diff
@@ -137,15 +86,15 @@
     We provide a single, secure connection to data from medical records, wearable sensors, testing services, and wellness devices and apps. Using our end-user authentication process, individuals can securely share their health data with any application or system, regardless of how that data was recorded, processed or stored."', employee_count: 20, tech_team_size: 20, website: 'https://www.humanapi.co/',logo: 'https://pbs.twimg.com/profile_images/378800000816901251/0dab64e7bd2cc59049520d7ca4649462_400x400.png', twitter: 'human_api', published: false
   }
 ]
-
 organization_list.each do |org|
   Organization.create( org )
   puts "Creating #{org[:name]}"
 end
-
+puts "21 Organizations have been created"
```
Comment (tkbeili) [inline db/seeds.rb]: Please replace `21` with `#{organization_list.count}` in case we add other ones

**Comment by tkbeili** [comment]
👍 Looks good

**Pull request merged**